service/src/main/java/com/epam/esm/service/service/impl/CertificateServiceImpl.java

```diff
@@ -181,11 +247,15 @@ public void addCertificateTag(int certificateId, Tag tag) throws ServiceExceptio
         Optional<Tag> tagOptional;
```

Comment:
there is no need to separate variable declaration and initialization

Reply:
Corrected